unicodetools/src/main/java/org/unicode/text/UCA/WriteCollationData.java

```diff
@@ -2133,6 +2133,10 @@ private static UCA buildCldrCollator(boolean addFFFx) {
             final int primary = ceCount < 1 ? 0 : CEList.getPrimary(output[0]);
             final int cat = Default.ucd().getCategory(i);
 
+            if (i == 0x20A7) {
+                // Starting with Unicode 15.1, U+20A7 PESETA SIGN sorts like "Pts".
```

Comment:
Note that UTC has not yet decided that ; we have made progress on discussing https://github.com/unicode-org/properties/issues/99, but we have not yet recommended anything.

Reply:
@eggrobin Noted, but I am trying to lean forward in DUCET on what seemed to be the consensus from that discussion. It is easy enough to pull it back if people dump on the change during beta review.